spanner/tests/system.py

```diff
@@ -258,12 +266,37 @@ def tearDown(self):
         for doomed in self.to_delete:
             doomed.delete()
 
-    def _check_row_data(self, row_data):
-        self.assertEqual(len(row_data), len(self.ROW_DATA))
-        for found, expected in zip(row_data, self.ROW_DATA):
+    def _assert_timestamp(self, value, nano_value):
+        self.assertIsInstance(value, datetime.datetime)
+        self.assertIn(value.tzinfo, (None, UTC))
```

Comment:
Why the ambiguity?  It seems like the caller should specify whether naive or UTC.